quickstart/101-devtest-labs/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "lab_id" {
+  value = azurerm_dev_test_lab.lab.id
+}
```

Comment:
As the password could be generated by `random_password`, we could add an output here:

```hcl
output "password" {
  sensitive = true
  value = local.password
}
```

Reply:
Fixed.

Reply:
@TomArcherMsft what is the intent of outputting the password? Marking it as sensitive will write it to the state but will not show it in any logging, etc.

Reply:
@grayzu Since the password is used as a Windows vm's password and this is a demo, we can assume that our users won't composite this root module with other modules, they just want to apply and then have a try, so this password is essential for them to try this vm.

Another approach is to provision a KeyValut, then store this password as a secret, only the user who has permission can retrieve this password, but the demo code would be a little more complex. WDYT?

Reply:
Ok, so since the end user needs to get this password, this should either:
- removed sensitive  = true so that the output will display the actual password
- go down the path suggested by longunmanb to save password to KeyVault

My $.02 would be to remove the sensitive = true since adding kv would add much more complication to this example.